src/App.scss

```diff
@@ -103,27 +103,11 @@ body {
     grid-row-start: 1;
     grid-row-end: 3;
     min-height: 12rem;
-    min-width: 19.5rem;
-}
-
-.App__scene-controls {
-    position: absolute;
-    z-index: 1;
-    bottom: 2rem;
-    right: 2rem;
-    display: grid;
-    grid-template-columns: 1fr;
+    min-width: 19.5rem; 
 }
 
 .App__scene-controls-group {
```

Comment:
Now this class is only used for the pen toggle switch, might be a good idea to change to more appropriate name?

Reply:
Agreed, I changed it to `App__ PenDownToggleSwitch-container`.